src/components/reusabel-ui/Logo.jsx

```diff
@@ -0,0 +1,39 @@
+import { styled } from "styled-components";
+import { theme } from "../../theme";
+
+export default function Logo() {
```

Comment:
```suggestion
export default function Logo({src}) {
```
Pour le rendre réutilisable.